hc/api/tests/test_ping.py

```diff
@@ -11,63 +11,96 @@ def setUp(self):
 
     def test_it_works(self):
         r = self.client.get("/ping/%s/" % self.check.code)
-        assert r.status_code == 200
+        self.assertEqual(200, r.status_code)
 
         self.check.refresh_from_db()
-        assert self.check.status == "up"
+        self.assertEqual("up", self.check.status)
 
         ping = Ping.objects.latest("id")
-        assert ping.scheme == "http"
+        self.assertEqual("http", ping.scheme)
 
     def test_it_handles_bad_uuid(self):
         r = self.client.get("/ping/not-uuid/")
-        assert r.status_code == 400
+        self.assertEqual(400, r.status_code)
```

Comment:
Is this change necessary? I think the old line and new line achieve the same thing.